automatic/ruby.portable/README.md

```diff
@@ -0,0 +1,8 @@
+# <img src="https://cdn.jsdelivr.net/gh/chocolatey-community/chocolatey-packages@aad7c15bfbec43c3716f8a82bc3af22e1a55579d/icons/ruby.svg" width="48" height="48"/> [ruby](https://chocolatey.org/packages/ruby)
+
+
+Ruby is a dynamic, open source programming language with a focus on simplicity and productivity. It has an elegant syntax that is natural to read and easy to write.
```

Comment:
Ruby is a dynamic, open source programming language focusing on simplicity and productivity. It has an elegant syntax that is natural to read and easy to write.